packages/base/src/util/TabbableElements.js

```diff
@@ -0,0 +1,50 @@
+
+import isNodeTabbable from "./isNodeTabbable.js";
+
+const getTabbableElements = node => {
+	const res = getTabbables(node.children);
+	// remove afterwards
+	console.log(res); // eslint-disable-line
+	return res;
+};
+
+const getLastTabbableElement = node => {
+	const tabbables = getTabbables(node.children);
+	return tabbables.length ? tabbables[tabbables.length - 1] : null;
+};
+
+const getTabbables = (nodes, tabbables) => {
+	const tabbablesNodes = tabbables || [];
+
+	Array.from(nodes).forEach(currentNode => {
+		if (currentNode.nodeType === Node.TEXT_NODE) {
+			return;
+		}
+
+		if (currentNode.shadowRoot) {
+			// If the current node has more than 1 child, the 1st child is the style tag,
```

Comment:
we might change the style tag to be the last one, and we might start rendering more style tags. a better way to handle future changes is to loop all children and skip ones where the tag name is `style`